src/main/java/org/zalando/nakadi/domain/BatchItem.java

```diff
@@ -58,4 +59,8 @@ public void updateStatusAndDetail(final EventPublishingStatus publishingStatus,
         response.setPublishingStatus(publishingStatus);
         response.setDetail(detail);
     }
+
+    public int getEventSize() {
+        return event.toString().getBytes(StandardCharsets.UTF_8).length;
```

Comment:
I think we should keep the original string `eventsAsString` in order to avoid calling `toString` which is too costly.